csrc/python_frontend/python_bindings.cpp

```diff
@@ -576,40 +570,47 @@ void initNvFuserPythonBindings(PyObject* module) {
   // This is the input version of define_vector
   fusion_def.def(
       "define_vector",
-      [](FusionDefinition& self,
-         size_t size,
-         PrimDataType dtype = DataType::Int) -> Vector {
-        return define_vector_fn(self, std::nullopt, size, dtype);
-      },
-      py::arg("size"),
-      py::arg("dtype") = DataType::Int,
-      py::return_value_policy::reference);
-  // This is the canonical version of define_vector that accepts either a
-  // nullptr or a vector of values to indicate either an input or a constant
-  // for use when printing out the associated Fusion Record.
-  fusion_def.def(
-      "define_vector",
-      [](FusionDefinition& self,
-         std::optional<std::vector<int64_t>> value,
-         size_t size,
-         PrimDataType dtype) -> Vector {
-        return define_vector_fn(self, value, size, dtype);
+      [](FusionDefinition& self, size_t size) -> Vector {
```

Comment:
nitpick: I almost want to ask for a DataType in the signature so that we are future proof. But that does also feel silly to have that there before having any real use case.

Reply:
I used to have it but I decided to tighten the interface as I had it defaulted to `DataType.Int` and don't have a reason to set it.